.github/workflows/dockerhub-description.yml

```
@@ -0,0 +1,21 @@
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - DOCKERHUB.md
+      - .github/workflows/dockerhub-description.yml
+jobs:
+  dockerHubDescription:
```

Please use snake case:
```suggestion
  docker-hub-description:
```

---

Agree that is better.